src/pharmpy/tools/amd/run.py

```diff
@@ -111,14 +106,21 @@ def run_amd(
     if search_space is None:
         if modeltype == 'pk_oral':
             search_space = (
-                'ABSORPTION([ZO,SEQ-ZO-FO]);'
-                'ELIMINATION([MM,MIX-FO-MM]);'
-                'LAGTIME();'
-                'TRANSITS([1,3,10],*);'
-                'PERIPHERALS(1)'
+                'ABSORPTION([ZO,SEQ-ZO-FO])\n'
+                'ELIMINATION([MM,MIX-FO-MM])\n'
+                'LAGTIME()\n'
+                'TRANSITS([1,3,10],*)\n'
+                'PERIPHERALS(1)\n'
+                'COVARIATE(@IIV, @CONTINUOUS, exp, *)\n'
+                'COVARIATE(@IIV, @CATEGORICAL, cat, *)'
             )
         else:
-            search_space = 'ELIMINATION([MM,MIX-FO-MM]);' 'PERIPHERALS([1,2])'
+            search_space = (
+                'ELIMINATION([MM,MIX-FO-MM])\n'
+                'PERIPHERALS([1,2])\n'
+                'COVARIATE(@IIV, @CONTINUOUS, exp, *)\n'
+                'COVARIATE(@IIV, @CATEGORICAL, cat, *)'
+            )
 
```

Comment:
Should there be defaults if the user only writes a search space for the structural model or vice versa? Previously, we set defaults for both and you could omit one of them to get a sensible default.

Reply:
That's a good idea. First solution I can think of is to have a way to decide whether a given MFL string contains any particular directive. For instance if there are no `COVARIATE` directives, then we should add the default ones.

Reply:
Yes, that sounds good! And if you only have `COVARIATE` directives we assume that no search space have been provided? Or maybe specify in respective module which features are included (now we could maybe use covariate vs non-covariate but I was thinking if this is expanded)?

Reply:
Each MFL statement has a type, so we can just have a list of types and if no statement has one of these types we add some default ones.

Reply:
Done!